packages/components/src/button-group/style.scss

```diff
@@ -15,5 +15,18 @@
 		&:last-child {
 			border-radius: 0 3px 3px 0;
 		}
+
+		// The focused button should be elevated so the focusring isn't cropped,
```

Comment:
I don’t think “focusring” is a real world and it tripped me up reading this. Maybe put a space between the words?